eng/Microsoft.Build.CentralPackageVersions/2.0.46/Sdk/Sdk.targets

```diff
@@ -130,7 +130,7 @@
   </ItemGroup>
 
   <Target Name="CheckPackageReferences"
-          Condition=" '$(EnableCentralPackageVersions)' != 'false' And '$(DesignTimeBuild)' != 'true' And @(PackageReference->Count()) > 0 ">
+          Condition=" '$(EnableCentralPackageVersions)' != 'false' And '$(DesignTimeBuild)' != 'true' And @(PackageReference->Count()) > 0 And '$(IsGeneratedSample)' != 'true'">
```

Comment:
Why is this needed? This file is a copy from another package and we should avoid modifying it if possible.